rclpy/src/rclpy/_rclpy.c

```diff
@@ -1434,6 +1434,10 @@ rclpy_destroy_entity(PyObject * Py_UNUSED(self), PyObject * args)
   } else if (0 == strcmp(entity_type, "timer")) {
     rcl_timer_t * timer = (rcl_timer_t *)p;
     ret = rcl_timer_fini(timer);
+  } else if (0 == strcmp(entity_type, "guard_condition")) {
+    rcl_guard_condition_t * guard_condition =
+      (rcl_guard_condition_t *)PyCapsule_GetPointer(pyentity, NULL);
```

Comment:
`PyCapsule_GetPointer(pyentity, NULL)` should be replaced by `p`.

Then this will also fit in one line.

Reply:
oh I copy pasted before rebasing :S